style.css

```diff
@@ -90,13 +96,14 @@ header {
     border-top: 1px solid #DDDDDD;
 }
 
-.intro-section-image {
-    max-width:100%;
-    max-height:100%;
+img {
+    width:100%;
+    height:100%;
 }
```

Comment:
i can see why you've done this - because it applies to all the images on your page - however it would be better to apply it specifically to the images you have, because this way if you decide to add another image it will automatically be 100% width and height which is a bit odd to have as the default.